tendenci/apps/events/models.py

```diff
@@ -2392,8 +2403,8 @@ def sub_events_by_datetime(self):
                 continue
 
             day = start_dt.date()
-            start_time = start_dt.strftime("%-I:%M %p")
-            end_time = sub_event_datetimes[start_dt].strftime("%-I:%M %p")
+            start_time = start_dt.strftime("%I:%M %p")
+            end_time = sub_event_datetimes[start_dt].strftime("%I:%M %p")
```

Comment:
looks like a typo snuck in something after we tested this... this is for the session check in grid